DESCRIPTION

```diff
@@ -11,7 +11,8 @@ Encoding: UTF-8
 Imports:
     dygraphs,
     odin (>= 0.0.7),
-    shiny
+    shiny,
+    grDevices
```

Comment:
this turns out not to be needed (and in general these should be alphabetical)

Reply:
Not sure I understand this, but if it is about grDevices, it seems needed:
```
devtools::check()
...

Status: 1 NOTE
See
  ‘/tmp/Rtmp3YzkI5/odin.ui.Rcheck/00check.log’
for details.


R CMD check results
0 errors | 0 warnings | 1 note 
checking R code for possible problems ... NOTE
transp: no visible global function definition for ‘col2rgb’
Undefined global functions or variables:
  col2rgb
Consider adding
  importFrom("grDevices", "col2rgb")
to your NAMESPACE file.```